### ISSUE TITLE
Possibly reword angle "slices" to "wedges"

### ISSUE THREAD

**Issue by terracoda** [phetsims/quadrilateral#295]
"pie slices" was the original idea, but we didn't want confusion between "pie" and mathematical "Pi", so we dropped "pie". "30 degree slices" in the hint is a little hard to find. And the numbered slices has been criticized by two people (low vision and blind populations).

@BLFiedler suggested "wedges".
- portions
- slivers
- chunks
Are other words that could be considered.

I will look at the help text and the object vales to see if a rename to "wedges" will work.

**Comment by terracoda** [comment]
I think it is safe to do a search and replace for "slice" with "wedge". This covers changing "slices" to "wedges", too, right?

**Comment by terracoda** [comment]
Note, I did a search and replace in the design document and things seem to look fine there.

**Comment by terracoda** [comment]
Need to adjust the phrasing in the checkbox's help text and context responses. A straight search and replace leaves those phrases a little awkward. The actual side and corner object responses are fine, though.

- [ ] Help text: Create shapes with or without help of 30 degree angle wedges and evenly-spaced side-length units.
- [ ] When checked: 30 degree angle wedges and evenly-spaced side units added.
- [ ] When unchecked: Angle wedges and side units hidden.
- [ ] Search and replace of "slice" with "wedge" and associated code clean-up.

I'll try to change the static stuff @jessegreenberg.

**Comment by terracoda** [comment]
I did the static stuff in 9eceffe18d89da68414c9b937c5998a13311bddf.

@jessegreenberg, leaving the search and replace for you.

**Comment by terracoda** [comment]
Nope, that's it. I'll give it a quick test run.

**Comment by terracoda** [comment]
The wedge descriptions sound fine in the overview. Will verify object responses after #306 is fixed.

**Comment by terracoda** [comment]
Wedges sound good. 

There is one object value/state, the 180 degree "angle flat" state that is described as "angle flat". Since the wedges are showing (not like in the right-angle state), I think it would be good to describe this state as "6 wedges".

I'll create a new issue for this and see if I can make the change myself. I don't want the change to affect the context response that says, "angle flat, two side look like one". I think that's a context response.
